mllib/src/main/scala/org/apache/spark/ml/evaluation/BinaryClassificationEvaluator.scala

```diff
@@ -31,15 +31,18 @@ import org.apache.spark.sql.types.DoubleType
  * Evaluator for binary classification, which expects two input columns: rawPrediction and label.
  */
 @Experimental
+@Since("1.2.0")
 class BinaryClassificationEvaluator(override val uid: String)
```

Comment:
Please annotate constructor and public params as well, see https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala#L130